### ISSUE TITLE
Cumpilation / Sexperience Cum Patch Miss-fires

### ISSUE THREAD

**Issue by vegapnk** [vegapnk/RJW-Genes#172]
User had Sexperience and Cumpilation.

Since today, [Sexperience 1.5.1.5](https://gitgud.io/amevarashi/rjw-sexperience/-/tags/1.5.1.5) has the Cum_Item only conditionally loaded if Cumpilation is not present. 
With both, the Patch fails. 

(And: The behaviour is currently not existant for the Cumpilation item)

[Hugslog](https://gist.github.com/HugsLibRecordKeeper/cd4bdf70365534f5f8c377430d8ddfee)

**Comment by vegapnk** [comment]
I got a patch-suggestion by Ryufas in the Discord: 

```xml
<?xml version="1.0" encoding="utf-8"?>
<Patch>

	<!-- 
		DevNote:
		There used to be an issue with a Gene-Mod Called "Cum-Addiction Gene". That mod just overwrote the
	complete UsedCondom.xml and Sexperience Cum.xml
		This would lead to two types of failures: 
		A) If this mod was before the other mod, changes were just overwritten
		B) If this mod was after, there was an issue in targetting the XML throwing a patch error
		Please if you write mods, do not just purge out XMLs. Thanks. 

		This patch was kindly mirrored after Infis Patch of UsedCondoms. 
	-->
	<Operation Class="PatchOperationFindMod">
		<mods>
			<li>RJW Sexperience</li>
		</mods>
		<match Class="PatchOperationConditional">
		    <xpath>Defs/ThingDef[defName="GatheredCum"]</xpath>
			<nomatch Class="PatchOperationFindMod">
			  <mods>
			    <li>Cumpilation</li>  
			  </mods>
			  <match Class="PatchOperationConditional">
			    <xpath>Defs/ThingDef[defName="Cumpilation_Cum"]/ingestible/outcomeDoers</xpath> 
				<nomatch Class="PatchOperationAdd">
				  <xpath>/Defs/ThingDef[defName="Cumpilation_Cum"]/ingestible</xpath>
				    <value>
					  <outcomeDoers>
						<li Class="RJW_Genes.IngestionOutcomeDoer_LifeForceOffset">
							<FertilinPerUnit>1</FertilinPerUnit>
						</li>
					  </outcomeDoers>
				    </value>
			    </nomatch>
			    <match Class="PatchOperationAdd">
				  <xpath>Defs/ThingDef[defName="Cumpilation_Cum"]/ingestible/outcomeDoers</xpath>
				    <value>
					  <li Class="RJW_Genes.IngestionOutcomeDoer_LifeForceOffset">
						<FertilinPerUnit>1</FertilinPerUnit>
					  </li>
				    </value>
				</match>
			  </match>
			</nomatch>
			<match Class="PatchOperationConditional">
			  <xpath>Defs/ThingDef[defName="GatheredCum"]/ingestible/outcomeDoers</xpath>
			  <nomatch Class="PatchOperationAdd">
				<xpath>/Defs/ThingDef[defName="GatheredCum"]/ingestible</xpath>
				<value>
					<outcomeDoers>
						<li Class="RJW_Genes.IngestionOutcomeDoer_LifeForceOffset">
							<FertilinPerUnit>1</FertilinPerUnit>
						</li>
					</outcomeDoers>
				</value>
			  </nomatch>
			  <match Class="PatchOperationAdd">
				<xpath>Defs/ThingDef[defName="GatheredCum"]/ingestible/outcomeDoers</xpath>
				<value>
					<li Class="RJW_Genes.IngestionOutcomeDoer_LifeForceOffset">
						<FertilinPerUnit>1</FertilinPerUnit>
					</li>
				</value>
			  </match>
		   </match>
		</match>
	</Operation>

</Patch>
```

**Comment by vegapnk** [comment]
Closed with 2.3.2
